index.bs

```diff
@@ -232,27 +232,130 @@ Non-Functional Requirements {#requirements-non-functional}
     APIs.
  
 
-Receiver Discovery {#discovery}
+Discovery with mDNS {#discovery}
 ===============================
 
-Issue(100): Incorporate material from the [mDNS](mdns.md) document.
+Endpoints may discover one another using
+[mDNS](https://tools.ietf.org/html/rfc6762) and
+[DNS-SD](https://tools.ietf.org/html/rfc6763).  To do so, endpoints must use the
+service name "_openscreen._udp.local".
+
+Endpoints must include DNS TXT records with the following keys and values: 
+
+- key "v" with value of the version of this protocol the endpoint understands.  
+  The current version of this protocol is "1". 
+
+- key "fp" with value of the certificate fingerpint of the endpoint.  The format
+  of the fingerprint is defined by [RFC 8122 section
+  5](https://tools.ietf.org/html/rfc8122#section-5), excluding the
+  "fingerprint:" prefix and including the hash function, space, and hex-encoded
+  fingerprint.  The fingerprint value also functions as an ID for the endpoint.
+  All endpoints must support the following hash functions: "sha-256", "sha-512".
+  Endpoints must not support the following hash functions: "md2", "md5". 
+
+- key 'time" with value of the last time the endpoint updated its
+  metadata.  The time value is represented as zero or a positive integer.  The
+  clock and timescale are chosen by the advertising endpoint, but are not
```

Comment:
nit: Would it make sense for this to just be specified as an arbitrary value that is _at least_ incremented whenever the metadata is updated, since it can't really be used as a time?

Reply:
Good idea.  I updated it thus.